src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -480,7 +480,8 @@ private NuGetPackageManager GetNuGetPackageManager(string solutionDirectory)
             {
                 if (!string.IsNullOrEmpty(exception.Message))
                 {
-                    nuGetProjectContext.Log(MessageLevel.Warning, exception.Message);
+                    var errorMessage = string.Format(Strings.RestoreErrorNU1000, exception.Message);
+                    nuGetProjectContext.Log(MessageLevel.Error, errorMessage);
```

Comment:
I don't think you should need to format the NU code into the message yourself. If you look at how other warnings/errors are created, we have helper methods. For example:
https://github.com/NuGet/NuGet.Client/blob/2247784eefe100547ee7af54f4ef5d6bf24c6d82/src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs#L464

Reply:
@zivkan 
Thank you for your suggestion. I used the helper method.